Prettier formatted and linted using eslint  on the entire project

Vandit1604 opened the pull request:
For uniformity across the project, formatted and linted the project using prettier and eslint.

krisstern commented:
HI @Vandit1604, can we please add a rule for enabling a blank line before the EOF?

krisstern commented:
Also @Vandit1604 , I think we should have a dicussion regarding the linting and formattting rules to use, in the spirit of open-source software development. Could you please do some research in this regard and make some recommendations as to which ones are appropriate for this project before we make a final decision as a team? I feel there has been no discussion prior to this and I have no idea why you are including certain rules and not the others.

Vandit1604 commented:
@krisstern I have pushed the prettier configuration for discussion. 
By default, prettier adds a blank line before EOF.

Apologies, I formatted the project assuming that I have added Ptettier and ESLint on the project, but I added it on the `site/` directory of https://github.com/Vandit1604/jenkins-docs/tree/main/site.

Should I reference the rules present here https://github.com/Vandit1604/jenkins-docs/blob/main/site/.prettierrc

krisstern commented:
HI @Vandit1604,
For the prettier rules you better check with us first before making the decision for the team. Could you please update the rules to the following?:

```json
{
    "trailingComma": "es5",
    "tabWidth": 4,
    "semi": false,
    "singleQuote": true,
    "bracketSpacing": true,
    "bracketSameLine": false,
    "endOfLine": "auto",
    "jsxSingleQuote": true
}
```

Vandit1604 commented:
> Could you please update the rules to the following

Done.

krisstern reviewed (approved):
LGTM
